devolving_music/lib/score_suite.py

```diff
@@ -1,3 +1,4 @@
+from cmath import inf
```

Comment:
`cmath.inf` is for complex numbers. Use `math.inf`.

Reply:
Why'd you resolve this?

Reply:
I thought that was in song_scores my bad i fixed that one too